README.md

```diff
@@ -31,18 +31,12 @@ Check out the [getting started guide](./docs/getting_started.md) for launching a
 
 This provider's versions are compatible with the following versions of Cluster API:
 
-|                     | Cluster API v1alpha3 (v0.7) | Cluster API v1alpha4 (v0.8) | Cluster API v1beta1 (v1.0) | Cluster API v1beta1 (v1.3) | Cluster API v1beta1 (v1.4) |
-|---------------------|:---------------------------:|:---------------------------:|:--------------------------:|:--------------------------:|:-------------------------:|
-| CAPV v1alpha3 (v0.7)(deprecated) |              ✓              |              ✓              |             ✓              |             ✓              |             ☓             |
-| CAPV v1alpha4 (v0.8)(deprecated) |              ☓              |              ✓              |             ✓              |             ✓              |             ☓             |
-| CAPV v1beta1 (v1.0) |              ☓              |              ☓              |             ✓              |             ✓              |             ☓             |
-| CAPV v1beta1 (v1.1) |              ☓              |              ☓              |             ☓              |             ✓              |             ☓             |
-| CAPV v1beta1 (v1.2) |              ☓              |              ☓              |             ☓              |             ✓              |             ☓             |
-| CAPV v1beta1 (v1.3, master) |              ☓              |              ☓              |             ☓              |             ✓              |             ☓             |
-| CAPV v1beta1 (v1.4) |              ☓              |              ☓              |             ☓              |             ✓              |             ☓             |
-| CAPV v1beta1 (v1.5) |              ☓              |              ☓              |             ☓              |             ✓              |             ✓             |
-| CAPV v1beta1 (v1.6) |              ☓              |              ☓              |             ☓              |             ✓              |             ✓             |
-| CAPV v1beta1 (v1.7) |              ☓              |              ☓              |             ☓              |             ✓              |             ✓             |
+|                                  | Cluster API v1alpha3 (v0.7) | Cluster API v1alpha4 (v0.8) | Cluster API v1beta1 (v1.0) | Cluster API v1beta1 (v1.3) | Cluster API v1beta1 (v1.4) | Cluster API v1beta1 (v1.5) |
```

Comment:
Could we also get rid of v1alpha3 and v1alpha4 from the top bar? Should make the whole thing easier to read.

Reply:
Oh of course ups

Reply:
Done, thx for catching it

Now the table looks okay